sample.env

```diff
@@ -51,20 +51,7 @@ GOOGLE_APPLICATION_CREDENTIALS='./resources/google/sample_gcp_service_account.js
 MONGO_DB_URI=
 MONGO_DB_NAME="ode"
 
-# Necessary Variables if running MongoDB locally with docker-compose-mongo.yml
-MONGO_ADMIN_DB_USER=root
-MONGO_ADMIN_DB_PASS=password
-MONGO_CV_MANAGER_DB_USER=root
-MONGO_CV_MANAGER_DB_PASS=password
-
-MONGO_IP=${DOCKER_HOST_IP}
-MONGO_DB_USER=root
-# This is the default password for the MongoDB container - MUST BE CHANGED FOR DEPLOYMENTS
-MONGO_DB_PASS=password
-MONGO_PORT=27017
-
 # If running MongoDB locally with docker-compose-mongo.yml uncomment the following line

```

Comment:
Should we remove this comment with the removal of the following line?

Reply:
This line is included in the develop sample.env, just re-added the docker-compose-mongo script